project.ini

```diff
@@ -1,11 +1,11 @@
+Files = cyclecloud_api-8.1.0-py2.py3-none-any.whl, job_submit_cyclecloud_centos_20.11.7-1.so, job_submit_cyclecloud_centos_19.05.8-1.so, job_submit_cyclecloud_centos8_20.11.7-1.so, job_submit_cyclecloud_suse_20.11.9-1.so, slurm-19.05.8-1.el7.x86_64.rpm, slurm-contribs-19.05.8-1.el7.x86_64.rpm, slurm-devel-19.05.8-1.el7.x86_64.rpm, slurm-example-configs-19.05.8-1.el7.x86_64.rpm, slurm-openlava-19.05.8-1.el7.x86_64.rpm, slurm-pam_slurm-19.05.8-1.el7.x86_64.rpm, slurm-perlapi-19.05.8-1.el7.x86_64.rpm, slurm-slurmctld-19.05.8-1.el7.x86_64.rpm, slurm-slurmd-19.05.8-1.el7.x86_64.rpm, slurm-slurmdbd-19.05.8-1.el7.x86_64.rpm, slurm-torque-19.05.8-1.el7.x86_64.rpm, job_submit_cyclecloud_ubuntu_20.11.7-1.so, job_submit_cyclecloud_ubuntu_19.05.8-1.so, slurm-contribs_19.05.8-1_amd64.deb, slurm-devel_19.05.8-1_amd64.deb, slurm-example-configs_19.05.8-1_amd64.deb, slurm-openlava_19.05.8-1_amd64.deb, slurm-pam-slurm_19.05.8-1_amd64.deb, slurm-perlapi_19.05.8-1_amd64.deb, slurm-slurmctld_19.05.8-1_amd64.deb, slurm-slurmd_19.05.8-1_amd64.deb, slurm-slurmdbd_19.05.8-1_amd64.deb, slurm-torque_19.05.8-1_amd64.deb, slurm_19.05.8-1_amd64.deb, slurm-20.11.7-1.el7.x86_64.rpm, slurm-contribs-20.11.7-1.el7.x86_64.rpm, slurm-devel-20.11.7-1.el7.x86_64.rpm, slurm-example-configs-20.11.7-1.el7.x86_64.rpm, slurm-openlava-20.11.7-1.el7.x86_64.rpm, slurm-pam_slurm-20.11.7-1.el7.x86_64.rpm, slurm-perlapi-20.11.7-1.el7.x86_64.rpm, slurm-slurmctld-20.11.7-1.el7.x86_64.rpm, slurm-slurmd-20.11.7-1.el7.x86_64.rpm, slurm-slurmdbd-20.11.7-1.el7.x86_64.rpm, slurm-torque-20.11.7-1.el7.x86_64.rpm, slurm_20.11.7-1_amd64.deb, slurm-pam-slurm_20.11.7-1_amd64.deb, slurm-contribs_20.11.7-1_amd64.deb, slurm-perlapi_20.11.7-1_amd64.deb, slurm-devel_20.11.7-1_amd64.deb, slurm-slurmctld_20.11.7-1_amd64.deb, slurm-example-configs_20.11.7-1_amd64.deb, slurm-slurmd_20.11.7-1_amd64.deb, slurm-libpmi_20.11.7-1_amd64.deb, slurm-slurmdbd_20.11.7-1_amd64.deb, slurm-openlava_20.11.7-1_amd64.deb, slurm-torque_20.11.7-1_amd64.deb, slurm-20.11.7-1.el8.x86_64.rpm, slurm-contribs-20.11.7-1.el8.x86_64.rpm, slurm-devel-20.11.7-1.el8.x86_64.rpm, slurm-example-configs-20.11.7-1.el8.x86_64.rpm, slurm-openlava-20.11.7-1.el8.x86_64.rpm, slurm-pam_slurm-20.11.7-1.el8.x86_64.rpm, slurm-perlapi-20.11.7-1.el8.x86_64.rpm, slurm-slurmctld-20.11.7-1.el8.x86_64.rpm, slurm-slurmd-20.11.7-1.el8.x86_64.rpm, slurm-slurmdbd-20.11.7-1.el8.x86_64.rpm, slurm-torque-20.11.7-1.el8.x86_64.rpm
```

Comment:
diff is hard to read.
one additional file `job_submit_cyclecloud_suse_20.11.9-1.so` 
inserted at fifth position